webapp/src/components/InputField/InputField.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+type Props = {
+    heading: {
+        id: string;
+        message: string;
+    };
+    tagType: string;
+    input: {
+        type?: string;
+        placeholder?: string;
+        maxLength?: number;
+        min?: number;
+        rows?: number;
+        defaultValue?: number;
+        onChange: (e: React.ChangeEvent<any>) => void;
```

Comment:
remove any and try to use proper type

Reply:
used "Element" type (base of all elements) was still getting TS error